reflector stopping fix

zigglobulin opened the pull request:
systemctl stop reflector.service fails when reflector is not started

picodotdev commented:
I tried systemctl stop with a stopped service in the system and the exit code was 0 so I suppose that systemctl stop not fails on this case.

Do you have the error message description?

zigglobulin commented:
You are right. I have added incorrect description in the commit message. This error occur when reflector is not installed in the system and reflector.service is not available in consequence.

I have updated the commit message.
